src/sql/parts/dashboard/pages/serverDashboardPage.contribution.ts

```diff
@@ -80,7 +80,7 @@ let defaultVal = [
 			'tasks-widget': [{ name: 'restore', when: '!mssql:iscloud' }, 'configureDashboard', 'newQuery']
 		},
 		gridItemConfig: {
-			sizex: 1,
```

Comment:
We shouldn't change these default config without pm input.

Reply:
Yes please get PM feedback.

Reply:
fixed.